writehat/templates/snippets/statusButtonGroup.html

```diff
@@ -0,0 +1,15 @@
+<div class="form-check form-check-inline">
+  <form id="status-radio">
+    <div class="btn-group btn-group-toggle" role="group" data-toggle="buttons">
+      <label for="inactive" class="btn btn-outline-primary">
+        <input type="checkbox" class="btn-check" id="inactive" name="status" value="inactive" autocomplete="off">Inactive
+      </label>
+      <label for="active" class="btn btn-outline-primary active">
+        <input type="checkbox" class="btn-check" id="active" name="status" value="active" checked autocomplete="off">Active
+      </label>
+      <label for="draft" class="btn btn-outline-primary">
+        <input type="checkbox" class="btn-check" id="draft" name="status" value="draft" autocomplete="off">Draft
```

Comment:
Can we set Draft to be checked by default as well? If someone saves a draft report and it's excluded by the default filter, they may suspect it didn't save correctly.